package.json

```diff
@@ -11,7 +11,7 @@
     ]
   },
   "scripts": {
-    "dev": "concurrently \"yarn foreach '@graduate/(common|api|frontend|api-client)' run dev\" \"yarn dev:proxy\"",
+    "dev": "concurrently \"yarn workspaces foreach --parallel --verbose --interlaced run dev\" \"yarn dev:proxy\"",
```

Comment:
We'd like to keep this as is for now because we're developing on a different version of yarn (`v3.2.0`) which supports the current command. We'll have to make this change later when we upgrade yarn to `v4.x`, but we'd like to do some workflow testing to make sure it doesn't break anything in prod beforehand.

Reply:
Think I've changed that. Can you check if it worked? Sorry for messing up with that, lol.